### ISSUE TITLE
New post error

### ISSUE THREAD

**Issue by Stoncs** [Luciferuz/kursovayaItransition#2]
# When you do new post, get 404 error
![image](https://user-images.githubusercontent.com/55064583/178580534-f09d817e-e474-44d4-a67c-b9bbe77e25b7.png)

**Comment by Stoncs** [comment]
upd: only if it with image

**Comment by Luciferuz** [comment]
yeah, amazon blocked my account due to a large debt =) so you can't upload pics now
